src/utils/addresses.ts

```diff
@@ -52,3 +52,14 @@ export const parsePrefixedAddress = (value: string): PrefixedAddress => {
 export const formatPrefixedAddress = (address: string, prefix?: string): string => {
   return prefix ? `${prefix}:${address}` : address
 }
+
+export const cleanInputValue = (value: string): string => {
+  const regex = new RegExp('(?:([a-z0-9]+):)?(0x[a-f0-9]{40})\\b', 'i')
+
+  const match = value.match(regex)
+  // if match, return the address with optional prefix
+  if (match?.length) return match[0]
```

Comment:
`match` will be null if no match, so no need to check the length.
```suggestion
  if (match?.length) return match[0]
```